pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```
@@ -127,6 +127,43 @@ spec:
             - name: socket-dir
               mountPath: /csi
         {{ end }}
+        {{ if .EnableCSIAddonsSideCar }}
+        - name: csi-addons
+          image: {{ .CSIAddonsImage }}
+          args :
+            - "--node-id=$(NODE_ID)"
+            - "--v=5"
+            - "--csi-addons-address=$(CSI_ADDONS_ENDPOINT)"
+            - "--controller-endpoint=$(POD_IP):4080"
+            - "--stagingpath=/var/lib/kubelet/plugins/kubernetes.io/csi/pv/"
```

```suggestion
            - "--stagingpath=/var/lib/kubelet/plugins/kubernetes.io/csi/pv/"
```

Make this configurable too 
refer https://github.com/rook/rook/blob/b69719ae775fa115e41ef8dd8b358c01711404ee/pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml#L70